Yosemite/Yosemite/Stores/Order/Order+CardPresentPayment.swift

```diff
@@ -37,10 +39,11 @@ extension Order {
         // * orders that have been partially refunded.
         // * orders where the merchant has applied a discount manually
         // * in general, all orders where we might want to capture a payment for less than the total order amount
-        let paymentViewModel = OrderPaymentDetailsViewModel(order: self)
-        return !paymentViewModel.hasBeenPartiallyCharged
+        let hasBeenPartiallyCharged = totalValue != netAmount
+        return !hasBeenPartiallyCharged
```

Comment:
So much better having this as simple logic than a side effect of making a new view model